src/content/docs/logs/get-started/get-started-log-management.mdx

```diff
@@ -44,36 +44,37 @@ Log management provides a way to connect your log data with the rest of your app
 
 ## Curious about costs? [#costs]
 
-Want to estimate the cost of reporting your logs? Use our [cost estimator tool](https://newrelic.com/blog/nerdlog/estimate-data-cost). 
+Want to estimate the cost of reporting your logs? Use our [cost estimator tool](https://newrelic.com/blog/nerdlog/estimate-data-cost). Also note that once you start using New Relic log management, it's easy to [adjust your logs ingest](/docs/data-apis/manage-data/manage-data-coming-new-relic). 
 
-## Bring in your logging data [#integrate-logs]
+## Report logs [#integrate-logs]
 
-To forward your log data to New Relic, you can use any of these options:
+To forward your log data to New Relic, you can use one or more of these options:
 
-* Let your app's APM agent forward log data directly to New Relic with [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context). No need to install or maintain additional third-party software!
-* Use our [guided install for infrastructure monitoring](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent) as a lightweight data collector for your apps' and hosts' logs.
-* Select from a wide range of [log forwarding plugins](/docs/logs/forward-logs/enable-log-management-new-relic/), including Amazon, Microsoft, Fluentd, Fluent Bit, Kubernetes, Logstash, and more.
-* Send your log data by using the [Log API](/docs/logs/log-api/introduction-log-api/) or [TCP endpoint](/docs/logs/log-api/use-tcp-endpoint-forward-logs-new-relic).
-* Use the OpenTelemetry SDK to send logs from your apps to an [OpenTelemetry collector](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs), which can forward them to New Relic via OTLP.
+* **Use our APM agents to report logs.** By default, our APM agents do two things: add metadata to your logs, which gives you logs-in-context (ability to see logs data in various relevant places in our platform) and forward your logs to New Relic. This is a popular option for DevOps teams and smaller organizations because it lets you easily report application logs, with no additional third-party solutions required. [Learn more about APM logs.](/docs/apm/new-relic-apm/getting-started/get-started-logs-context)
```

Comment:
I see it is logs-in-context here but I don't think there are dashes anywhere else before now. Is there a reason this one is written this way?